src/Utils/Utils.java

```diff
@@ -50,7 +50,14 @@ public static void runCalcThread() {
         Thread thread = new Thread(() -> {
             GUI.displayAsLoading(true);
 
-            String resultAsString = "" + Calculations.convertCurrencies("EUR", "USD", 29.99);
+            /*
+             * 
+             * TODO: Code Optimization (GUI.result etc.)
+             * 
+             */
+            String resultAsString = ""
+                    + Calculations.convertCurrencies(GUI.result, GUI.result2, GUI.inputValueAsDouble);
+
             GUI.setOuput("Ergebnis ist " + resultAsString.replace('.', ',') +
```

Comment:
Würde hier auf jeden Fall dann den Output verbessern wenn du schon dabei bist. Irgendwas was sagt hey du hast die und die währung ausgewählt, und x euro sind y usd und rausgefunden in z millisekunden. Checkst du? 👀

Reply:
@Syhnx vergessen??